src/List.js

```diff
@@ -65,6 +65,14 @@ export class List extends IndexedCollection {
     return node && node.array[index & MASK];
   }
 
+  has(index) {
```

Comment:
This seems identical to the `has` implementation of `IndexIterable` which is already in the prototype chain.